web/themes/custom/usagov/templates/block--usagov-taxonomy-links.html.twig

```diff
@@ -0,0 +1,35 @@
+{#
+/**
+ * @file
+ * Taxonomy links layout. This is used for a block with the "Bookmarks" category and is
+ * not in the block layout; instead it is rendered into a variable by the usa_twig_vars_preprocess
+ * function so that bookmarks can be used to generate Taxonomy information for the HTML head
+ * of a page.
+ */
+#}
+{% set nodeID = node.nid[0].value %}
+{% set path = drupal_url('node/'~nodeID) %}
+{% set links = content['#links'] %}
+{# Since six entries are wanted, we'll probably need a "for i in 0..5" loop #}
+{# see https://twig.symfony.com/doc/3.x/tags/for.html #}
+{# Code is not indented to keep proper formatting when viewing page source for datalayer #}
+
+{% set listLimit = 5 %}
+{% set counter = 0 %}
+{% for i in 0..listLimit %}
+{% set counter = counter + 1 %}
+{% if links[i] %}
+{% set link = links[i] %}
+{% endif %}
+      "Taxonomy_Text_{{ counter }}": "{{ link.getText() }}",
+{% endfor %}
+{% set counter = 0 %}
+{% for i in 0..listLimit %}
+{% set counter = counter + 1 %}
+{% if links[i] %}
```

Comment:
If you change this to `{% if links[i] and links[i].getUrl().toString() %}` it will work. 
I'm reviewing even though you didn't say this is ready, so you might already have noticed, the current page is coming up with an empty link right now. For example, on /government-loan I see this: 

```
"Taxonomy_Text_1": "Home",
"Taxonomy_Text_2": "Money and credit",
"Taxonomy_Text_3": "Government grants and loans",
"Taxonomy_Text_4": "How to get a government loan",
"Taxonomy_Text_5": "How to get a government loan",
"Taxonomy_Text_6": "How to get a government loan",
"Taxonomy_URL_1": "/",
"Taxonomy_URL_2": "/money",
"Taxonomy_URL_3": "/government-grants-and-loans",
"Taxonomy_URL_4": "",
"Taxonomy_URL_5": "/government-loan",
"Taxonomy_URL_6": "/government-loan",
```
It's because the breadcrumb for the current page does have a Link object (I didn't realize this either), but the string representation of its URL is blank. 

I always check whether it's okay to string together something like ` links[i].getUrl().toString()` in an `if` clause like this -- might `links[i].getUrl()` return null, and we'll get an error trying to run `toString()`? In this case Drupal's docs tell us the getUrl() method on a link always returns a \Drupal\Core\Url object, so we can safely do this.